xml/security_ldap_ca.xml

```diff
@@ -34,56 +34,58 @@
     certificate, and a root certificate.
   </para>
  <procedure>
-  <para>
-  Before you can import an existing private key and certificate into the NSS
-  database, you need to create a bundle of the private key and the server
-  certificate. This results in a <filename>*.p12</filename>
-  file.
-  </para>
   <important>
-  <title><filename>*.p12</filename> file and friendly name</title>
-  <para>
-   When creating the PKCS12 bundle, you must encode <literal>Server-Cert</literal>
-   as the friendly name in the <filename>*.p12</filename> file.
-   Otherwise the TLS connection will fail, because the &ds389; searches for
-   this exact string.
-  </para>
   <para>
-    The friendly name cannot be changed after you
-    import the <filename>*.p12</filename> file into the NSS
-    database.
+  The Mozilla NSS toolkit uses nicknames for certificates in the certificate store.
+  It is hard coded inside 389 Directory Server that the server certificate uses the nickname "Server-Cert".
```

Comment:
This isn't true. We *advise* you use Server-Cert because it makes a lot of things easier, but it's not hardcoded. It's important to not make assumptions here.

Reply:
Noted and